docs/userguide.rst

```diff
@@ -96,6 +96,31 @@ so that a hard copy of the link target is uploaded to the cloud
 storage workspace. The link is not maintained throughout the
 workflow execution.
 
+
+Deleting analysis assets
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+The deletion of files contained in the analysis workspace is possible through
+the ``remove`` command:
+
+.. code-block:: console
+
+   $ reana-client remove file1
```

Comment:
As we usually do, please use the "my..." convention to make it more easily understandable to users.

E.g. `mydata.csv`, `mycode.C` as used elsewhere,  or even `mytmp/myfiltereddata.csv` to indicate the temporary nature of things to be removed...